pkg/oc/cli/observe/observe.go

```diff
@@ -274,6 +275,7 @@ func NewCmdObserve(fullName string, f kcmdutil.Factory, streams genericclioption
 	// additional debug output
 	cmd.Flags().BoolVar(&o.noHeaders, "no-headers", false, "If true, skip printing information about each event prior to executing the command.")
```

Comment:
Umm... since you are changing flag name, what do you think about this one? `--no-headers` is used elsewhere for something completely different. I'd propose changing this to `--debug` or something like that, wdyt?

Reply:
Although, honestly, why can't we just use `--loglevel` for that? @smarterclayton objections?